Verifying sv/ldv-challenges/linux-3.8-rc1-32_7a-drivers--net--ethernet--broadcom--tg3.ko-ldv_main0_sequence_infinite_withcheck_stateful_true-unreach-call.cil.out.c
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_error
    ldv_atomic_dec_and_mutex_lock_mutex_of_device
    <end>
    ldv_mutex_lock_killable_mutex_of_device
    <end>
    ldv_mutex_lock_interruptible_mutex_of_device
    <end>
    ldv_atomic_dec_and_mutex_lock_lock
    <end>
    ldv_mutex_trylock_lock
    <end>
    ldv_mutex_lock_killable_lock
    <end>
    ldv_mutex_lock_interruptible_lock
    <end>
    ldv_mutex_unlock_mutex_of_device
      ldv_mutex_unlock_5
      <end>
    ldv_mutex_trylock_mutex_of_device
      ldv_mutex_trylock_4
      <end>
    ldv_mutex_lock_mutex_of_device
      ldv_mutex_lock_3
      <end>
    ldv_mutex_unlock_lock
      ldv_mutex_unlock_2
      <end>
    ldv_mutex_lock_lock
      ldv_mutex_lock_1
      <end>
    ldv_check_final_state
      main
      <end>
    ldv_check_final_state
      main
      <end>
